bridge/hardhat.config.js

```diff
@@ -11,7 +11,8 @@ require("@nomiclabs/hardhat-etherscan");
 const fs = require('fs');
 const chains = require('./hardhat/helper/chains');
 const MNEMONIC = fs.existsSync('./mnemonic.key') ? fs.readFileSync('./mnemonic.key', {encoding: 'utf8'}) : ''; // Your metamask's recovery words
-const INFURA_PROJECT_ID = fs.existsSync('./infura.key') ? fs.readFileSync('./infura.key', {encoding: 'utf8'}) : ''; // Your Infura project ID
+const ETHERESCAN_KEY = fs.existsSync('./etherscan.key') ? fs.readFileSync('./etherscan.key', {encoding: 'utf8'}) : ''; // Your metamask's recovery words
```

Comment:
I'd remove the `Your metamask's recovery words` comment from here

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/276/commits/2fc10b0d316242fb067c4ac2e94684b00ce0c64f